<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionMenu</class>
 <widget class="QWidget" name="FunctionMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Меню функций</string>
  </property>
  <widget class="QPushButton" name="hordMethodButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Хорды</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageEmbedButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Картинка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vigenereCipherButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Шифр Виженера</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sha384Button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>SHA-384</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
